template/pkg/flagset/flagset.go

```diff
@@ -82,14 +82,14 @@ func ServerWithConfig(cfg *config.Config) []cli.Flag {
 		},
 		&cli.StringFlag{
 			Name:        "tracing-service",
-			Value:       "ocs",
+			Value:       "{{ trimPrefix Name }}",
```

Comment:
You're missing an argument here, did you mean:

    {{ trimPrefix Name `ocis-` }} ?

Reply:
done